Issue template for bug report.

laxmikantbpandhare opened the pull request:
As discussed in today's community meeting, added issue template for bug report.

metacosm commented on .github/ISSUE_TEMPLATE/bug-report.md:
```diff
@@ -0,0 +1,55 @@
+
+<!-- The type of cluster used for testing/deployment, ex. "vanilla", "OpenShift" -->
+
+`$ java-operator-sdk version`
```
This implies running a CLI which doesn't exist…

laxmikantbpandhare replied:
So, how one can check about the version he is using. `pom.xml` file?

laxmikantbpandhare replied:
@metacosm  - Modified the instructions. Asked user to verify `java-operator-sdk` version from `pom.xml` file.

metacosm commented on .github/ISSUE_TEMPLATE/bug-report.md:
```diff
@@ -0,0 +1,55 @@
+
+<!-- Insert the output of `operator-sdk version` here. -->
+
+`$ java version`
```
```suggestion
`$ java -version`
```

laxmikantbpandhare replied:
Modified.

metacosm reviewed (changes requested):
Please rebase your PR so that all commits follow the conventional commit message format: https://www.conventionalcommits.org/en/v1.0.0/

laxmikantbpandhare commented:
> Please rebase your PR so that all commits follow the conventional commit message format: https://www.conventionalcommits.org/en/v1.0.0/

@metacosm I updated all the messages with `style` tag as we are adding template/styling for our issue/bug creation process. I squashed all the commits as well.

metacosm commented:
Thank you!
